stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/event/handler/internal/EventHandlerImpl.kt

```diff
@@ -615,6 +616,15 @@ internal class EventHandlerImpl(
                 }
         }
 
+        // handle events for active threads
+        sortedEvents.filterIsInstance<HasMessage>()
+            .groupBy { it.message.parentId ?: it.message.id }
+            .forEach { (messageId, events) ->
+                if (logic.isActiveThread(messageId)) {
```

Comment:
`logic.isActiveThread(messageId)` can be moved to a `filter` function.

Reply:
I applied it!